src/Questionnaire/Questionnaire.js

```diff
@@ -27,14 +30,27 @@ export default function Questionnaire({ testIds }) {
 
   return (
     <>
-      <h1>Fragebogen {questionRound + 1} / 2</h1>
-      <p>"{personalityStyleData[currentTestIndex]?.name}"</p>
+      <img alt="" src="/img/questionnaire-intro.svg" />
+      <HeadlineUnderline>
+        <h1>Fragebogen {questionRound + 1} / 2</h1>
+      </HeadlineUnderline>
       <h2>
         Frage {currentQuestionIndex + 1} / {questions.length}
       </h2>
       <h3>{questions[currentQuestionIndex]}</h3>
-      <button onClick={() => handleAnswer(false)}>(Eher) Nein</button>
-      <button onClick={() => handleAnswer(true)}>(Eher) Ja</button>
+
+      <ButtonRow>
+        <Button
+          btnType="white"
+          width="47.5"
```

Comment:
Why do you define the width here and not in a styled component part?

Reply:
When I change the width I also need to change "display" to "inline". I couldn't figure out how to do that with a styled component, so I put that logic into my Button component.